admin/services/group.service.js

```diff
@@ -83,6 +83,9 @@ groupService.update = async (id, group, schoolId) => {
   if (!(pupils instanceof Array)) {
     pupils = Object.values(pupils)
   }
+  if (typeof group.name === 'string') {
```

Comment:
Is there a case where this could come as a non-string value?

Reply:
Not really possible, since in both cases we're using a validator that name exists and is a string.

Reply:
@danwilliams wouldn't that make the if condition redundant then?

Reply:
Resolving since a newer commit covered that question